src/features/search/components/NativeCategoryValue/NativeCategoryValue.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import styled from 'styled-components/native'
+
+import { NativeCategoryIdEnumv2, SubcategoriesResponseModelv2 } from 'api/gen'
+import { getNativeCategoryFromEnum } from 'features/search/helpers/categoriesHelpers/categoriesHelpers'
+import { Typo } from 'ui/theme'
+
+interface NativeCategoryValueProps {
+  nativeCategoryId: NativeCategoryIdEnumv2
+  data?: SubcategoriesResponseModelv2
+}
+
+export const NativeCategoryValue = ({ nativeCategoryId, data }: NativeCategoryValueProps) => {
+  const { value } = getNativeCategoryFromEnum(data, nativeCategoryId) || {}
```

Comment:
Si tu suis ma recommandation de dessus tu pourrais avoir 
```suggestion
  const { data } = useSubcategories()
  const { value } = getNativeCategoryFromEnum(data, nativeCategoryId) || {}
```